src/main/java/uk/gov/hmcts/reform/roleassignment/util/SecurityUtils.java

```diff
@@ -23,46 +23,42 @@ public SecurityUtils(final AuthTokenGenerator authTokenGenerator) {
 
     public HttpHeaders authorizationHeaders() {
         final HttpHeaders headers = new HttpHeaders();
-        headers.add(Constants.SERVICE_AUTHORIZATION, authTokenGenerator.generate());
+        headers.add(Constants.SERVICE_AUTHORIZATION2, authTokenGenerator.generate());
+        headers.add(HttpHeaders.AUTHORIZATION, getUserAuthorizationHeaders());
+        return headers;
+    }
 
+    public String getUserAuthorizationHeaders() {
         if (SecurityContextHolder.getContext().getAuthentication() != null) {
-            final ServiceAndUserDetails serviceAndUser = (ServiceAndUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+            final ServiceAndUserDetails serviceAndUser = (ServiceAndUserDetails) SecurityContextHolder.getContext()
+                                                                                                      .getAuthentication()
+                                                                                                      .getPrincipal();
             if (serviceAndUser.getPassword() != null) {
-                headers.add(HttpHeaders.AUTHORIZATION, serviceAndUser.getPassword());
+                return serviceAndUser.getPassword();
             }
         }
-        return headers;
-
+        return null;
     }
 
-    public HttpHeaders serviceAuthorizationHeaders() {
-        final HttpHeaders headers = new HttpHeaders();
-        headers.add(Constants.SERVICE_AUTHORIZATION, authTokenGenerator.generate());
-        return headers;
-
+    public String getServiceAuthorizationHeader() {
+        return authTokenGenerator.generate();
     }
 
     public String getUserId() {
-        final ServiceAndUserDetails serviceAndUser = (ServiceAndUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+        final ServiceAndUserDetails serviceAndUser = (ServiceAndUserDetails) SecurityContextHolder.getContext()
+                                                                                                  .getAuthentication()
+                                                                                                  .getPrincipal();
         return serviceAndUser.getUsername();
     }
 
-    public String getUserToken() {
-        final ServiceAndUserDetails serviceAndUser = (ServiceAndUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
-        return serviceAndUser.getPassword();
-    }
-
-    public String getServiceId() {
-        final ServiceAndUserDetails serviceAndUser = (ServiceAndUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
-        return serviceAndUser.getServicename();
-    }
-
```

Comment:
we need this method to retrieve the client id from the assignment request token

Reply:
Reverted the code and put back this method.